index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,400;0,500;0,700;1,100;1,400;1,700&display=swap"
+    rel="stylesheet">
+  <title>Portfolio</title>
+</head>
+
+<body>
+  <div class="wrapper">
+    <header>
+      <nav>
+        <a href="#">
+          <div class="logo">
+            My Portfolio
+          </div>
+        </a>
+        <div class="menu">
+          <a href="#">
+            <img src="src/icons/hamburger.svg" alt="">
+          </a>
+        </div>
+      </nav>
+    </header>
+
+    <main>
+      <h2 class="main-title">I'm Sasan<br>Glad to see you!</h2>
```

Comment:
- [ ] The main title of a page should always be an `<h1>` tag. If you want google (and other search engines) to know about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.🥂